SimPEG/electromagnetics/frequency_domain/simulation.py

```diff
@@ -552,6 +613,7 @@ def getRHSDeriv(self, freq, src, v, adjoint=False):
 ###############################################################################
 
 
+@with_property_mass_matrices("permittivity")
```

Comment:
```suggestion
```

You should be able to remove this as you are not doing anything like `MePermitivity`